### ISSUE TITLE
So did anyone succeeded in building it?

### ISSUE THREAD

**Issue by nogitsune-youkai** [StrongPC123/Far-Cry-1-Source-Full#1]
Title says it all

**Comment by nogitsune-youkai** [comment]
Apparently no...

**Comment by Martinligabue** [comment]
given issue #3 I get it's possible

**Comment by Yeeler** [comment]
No.  It didn't build with Visual Studio 2022.

**Comment by tigros** [comment]
just for fun i did get it to compile but no thanks to this repo, there's missing files like CAISystem.cpp etc, i think he has the worst antivirus known to man or something.. so look for FarCry 1.34 Complete on archive.org instead.

I used a WinXP SP3 VM with VS.Net 2003 installed, there were a few problems but not too bad.

Also have to get the original game for the data + w/e else missing and copy it.

The cool thing is I copied it to host and can debug it with modern VS thanks to attach to process. Doesn't run in VM, like most games.

I tried compiling it with VS 2017 at first but way too many problems.

Also it runs a compatibility thing "rundll32.exe C:\Windows\system32\gameux.dll,GameUXShim..." which gets stuck, trick is to kill farcry.exe, leave rundll32 running and start up farcry again.

**Comment by StrongPC123** [comment]
Yes, on discord server https://discord.com/invite/far-cry-1-community-472740869208801292 guys successfully build it.

**Comment by StrongPC123** [comment]
> No. It didn't build with Visual Studio 2022.

this build adapted for 2003, needed port on newest visuals

**Comment by Historical-17** [comment]
Yes , but not me :)
